fluent-syntax/src/stream.js

```diff
@@ -24,6 +26,7 @@ export class ParserStream {
     }
 
     this.index++;
+    this.checkLineBreak();
```

Comment:
> I would much prefer to calculate the line at the time we encounter an error.

In the PR, I do that for every entry, valid or not. This will allow to easily print warnings for valid entries the same way errors are displayed right now. Do you think it's not worth it?